### PR TITLE
Updated NEWS for summer 2021.

### PR DESCRIPTION
**Pull request by whitwham**
Please take a look for errors, omissions or entirely fictional updates.

This will be kept updated until we are ready for a release.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,21 +1,75 @@
...
- * samtools addreplacerg now allows for updating (replacing) an existing
-   `@RG` line in the output header, if a new `@RG` line is provided in the
-   command line, via the -r argument. The update still requires the user's
-   approval, which can be given with the new -w option.
+   remains supported. Thanks to David McGaughey and John Marshall.
+   (#1434)
+   
+ * Replace fopen with stat for file exist checks.  Prevents hanging when used
+   on a named pipe.
+   (#1438; fixes #1438)
```
Comment (valeriuo) [inline NEWS:55]: `fixes #1437`

Comment (whitwham) [reply NEWS]: Done.

File: NEWS [hunk 2]
```diff
@@ -1,21 +1,75 @@
...
+  
+ * Supports multiple refs/chroms from a BED file in ampliconclip, ampiliconstats
+   and plot-ampliconstats.
+   (#1410 and #1417; fixes #1396 and #1418)
+  
+ * Stopped samtools cat from outputting multiple CRAM EOF markers.
+   (#1422)
+
+ * Made samtools coverage ASCII mode produce true ASCII output.  Previously it
+   would produce UTF8 characters.
```
Comment (jmarshall) [inline NEWS:32]:
```suggestion
   would produce UTF-8 characters.
```

**Comment by jmarshall** [comment]
Now that it's been merged 🎉 it would be useful to record when `view` gained long options:

```
 * samtools view now has long option equivalents for most of its
   single-letter options. Thanks to John Marshall. (#1442)
```

**Comment by daviesrob** [comment]
I've added a couple of extra updates, to reorder items so that related ones are closer to each other, and to make a few other minor adjustments to the wording.

File: NEWS [hunk 3]
```diff
@@ -1,21 +1,98 @@
...
+ * Added options to set and clear flags to samtools view.  Along with the
+   existing remove aux tags this gives the ability to remove mark duplicate
+   changes (part of #1358)
+   (#1441)
+
+ * samtools view now has long option equivalents for most of its  single-letter
+   options. Thanks to John Marshall.
+   (#1442)
+
+ * A new tool, samtools import, had been added.  It reads one or more FASTQ
```
Comment (jmarshall) [inline NEWS:20]:
```suggestion
 * A new tool, samtools import, has been added.  It reads one or more FASTQ
```

File: NEWS [hunk 4]
```diff
@@ -1,21 +1,98 @@
...
+   target regions) when unmapped reads were present.
+   Thanks to @vinimfava (#1451), @JingGuo1997 (#1457) and Ramprasad Neethiraj
+   (#1460) for reporting the respective issues.
+
+ * Added options to set and clear flags to samtools view.  Along with the
+   existing remove aux tags this gives the ability to remove mark duplicate
+   changes (part of #1358)
+   (#1441)
+
+ * samtools view now has long option equivalents for most of its  single-letter
```
Comment (jmarshall) [inline NEWS:16]:
```suggestion
 * samtools view now has long option equivalents for most of its single-letter
```

File: NEWS (hunk 3 above)
Comment (daviesrob) [reply NEWS]: Fixed now, thanks.

**Pull request merged**